libtremor/VITABUILD

```diff
@@ -2,11 +2,13 @@ pkgname=libtremor
 pkgver=9999
 pkgrel=1
 url="https://xiph.org/"
-source=("git+https://gitlab.xiph.org/xiph/tremor.git#commit=293fd1c0")
+source=("git+https://gitlab.xiph.org/xiph/tremor.git#branch=lowmem")
```

Comment:
Could you change this to use the commit number instead? Otherwise we'll be shipping updated versions without realizing later.